evm/p2p/les.py

```diff
@@ -223,9 +224,20 @@ class Proofs(Command):
     structure = [
         ('request_id', sedes.big_endian_int),
         ('buffer_value', sedes.big_endian_int),
-        ('nodes', sedes.CountableList(sedes.CountableList(sedes.raw))),
+        ('proofs', sedes.CountableList(sedes.CountableList(sedes.raw))),
     ]
 
+    def decode_payload(self, rlp_data: bytes) -> _DecodedMsgType:
+        decoded = super().decode_payload(rlp_data)
+        # This is just to make Proofs messages compatible with ProofsV2, so that LightChain
+        # doesn't have to special-case them. Soon we should be able to drop support for LES/1
+        # anyway, and then all this code will go away.
```

Comment:
It's not clear to me which of the following is LES/1 and which is LES/2.  Can you explicitely label them.

Reply:
This method is under the Proofs class, and there's another class named ProofsV2, so I thought that'd make it clear enough. I guess it wasn't clear enough here because of lack of context?

Reply:
Sorry, I mis-understood what I was looking at.  Disregard.